\documentclass[fleqn,10pt]{wlscirep}

%% define a new envrionment which marries longtable with tabulary
% from http://tex.stackexchange.com/questions/78075/multi-page-with-tabulary (see there for usage)
% with modifications taken from the ltxtable package to make captions work
\makeatletter

\def\ltabulary{%
	\def\endfirsthead{\\}%
	\def\endhead{\\}%
	\def\endfoot{\\}%
	\def\endlastfoot{\\}%
	\def\tabulary{%
		\def\TY@final{%
			\def\endfirsthead{\LT@end@hd@ft\LT@firsthead}%
			\def\endhead{\LT@end@hd@ft\LT@head}%
			\def\endfoot{\LT@end@hd@ft\LT@foot}%
			\def\endlastfoot{\LT@end@hd@ft\LT@lastfoot}%
			\longtable}%
		\let\endTY@final\endlongtable
		\TY@tabular}%
	\dimen@\columnwidth
	\advance\dimen@-\LTleft
	\advance\dimen@-\LTright
	\tabulary\dimen@}

\def\endltabulary{\endtabulary}

\makeatother


% Packages
\usepackage{cleveref}
\usepackage{booktabs}
\usepackage{tabulary, booktabs, longtable, ltcaption, ltabptch}
%\usepackage[referable]{threeparttablex}
\usepackage{gensymb}
\usepackage{subcaption}
\captionsetup{compatibility=false}

% Set up of the bibliography style
\RequirePackage[backend=biber, style=authoryear, citestyle=authoryear, sorting=nty, natbib=true, doi=true, isbn=true, url=false]{biblatex}


\title{Network analysis of the hominin origin of Herpes Simplex virus 2 from fossil data}
%\shorttitle{Hominin origin of HSV2}
\author[1,2]{Simon Underdown}
\author[3]{Krishna Kumar}
\author[4,5,*]{Charlotte Houldcroft}
\affil[1]{Human Origins and Palaeoenvironmental Research Group (HOPE), Department of Anthropology \& Geography, Oxford Brookes University, Oxford, OX3 0BP, UK.}
\affil[2]{Leverhulme Centre for Human Evolutionary Studies, University of Cambridge, Henry Wellcome Building, Fitzwilliam Street, Cambridge, CB2 1QH, UK.}
\affil[3]{Computational Geomechanics, Cambridge University Engineering Department, Trumpington Street, Cambridge, CB2 1PZ, UK.}
\affil[4]{Division of Biological Anthropology, Department of Archaeology \& Anthropology, University of Cambridge, Cambridge, CB2 3QG, UK.}
\affil[5]{McDonald Institute for Archaeological Research, University of Cambridge, Downing Street, Cambridge, CB2 3ER, UK.}

\affil[*]{ch504@cam.ac.uk}

\keywords{network analysis; human evolution; infectious disease; virology; archaeology}

\begin{abstract}
Herpes simplex virus 2 is a human herpesvirus found worldwide that causes genital lesions and more rarely causes encephalitis. This pathogen is most common in Africa, and particularly in central and east Africa, an area of particular significance for the evolution of modern humans. Unlike HSV1, HSV2 has not simply co-speciated with humans from their last common ancestor with primates. HSV2 jumped the species barrier between 1.4 and 3 MYA, most likely through intermediate but unknown hominin species.
  

In this paper, we use probability-based network analysis to determine the most probable transmission route between intermediate hosts of HSV2, from the ancestors of chimpanzees to the ancestors of modern humans, using paleo-environmental data on the distribution of African tropical rainforest over the last 3 million years and data on the age and distribution of fossil species of hominin present in Africa between 1.4 and 3 MYA. Our model identifies \textit{Paranthropus boisei} as the most likely intermediate host of HSV2, while \textit{Homo habilis} may also have played a role in the initial transmission of HSV2 from the ancestors of chimpanzees to \textit{P. boisei}.
\end{abstract}


\addbibresource{references.bib}

\begin{document}

\flushbottom
\maketitle
%
\thispagestyle{empty}

\textbf{Keywords}
network analysis; human evolution; infectious disease; epidemiology; archaeology

\section*{Introduction}
Herpes simplex virus 2 (HSV2) is a sexually transmitted human pathogen that causes genital lesions and, rarely, encephalitis (eg~\citet{Tang2003}), and is associated with increased risk of HIV acquisition~\citep{Freeman2006}. After primary infection, the virus adopts a life cycle of latency punctuated by periods of lytic replication when new hosts can be infected through genital contact. The virus is related to the human oral pathogen herpes simplex virus 1 (HSV1). Both HSV1 and HSV2 are alphaherpesviruses, which are found in many primates~\citep{Wertheim2014}. HSV1 primarily causes infection, and sporadic lesions, within the oral cavity and establishes latency in the trigeminal ganglia, while HSV-2 is associated with infection of the genitalia and surrounding skin, and establishes latency in the sacral ganglia~\citep{Whitley2007}. Both simplex viruses can infect either body cavity, although there is viral shedding data from co-infected individuals to suggest that HSV1 is a more successful oral and HSV2 a more successful genital pathogen~\citep{Kim2006}. The genetic basis of this difference in tropism isn't fully understood: some studies of recombinant HSV1 x HSV2 strains have highlighted the importance of the latency-associated transcript (LAT) in how successfully the two simplex viruses reactivate from latency within different nerve types~\citep{Bertke2007}. HSV1 and 2 also differ in the length of the glycoprotein G (US4) open reading frame, which may play a role in tropism~\citep{Baines2007}.

HSV2 was originally thought to have co-speciated with humans when our lineage diverged from that of the ancestors of chimpanzees and bonobos (anc-chimps). Comparisons of the HSV1, HSV2 and chimpanzee herpes virus 1 (ChHV1) genomes~\citep{Tang2003} suggest that HSV2 is more closely related to ChHV1 than HSV1~\citep{Wertheim2014}. This analysis also found that ChHV1 and HSV2 diverged from one another between 1.4 and 3 MYA, and the authors inferred that an unknown hominin (or hominins) was infected with HSV2 before it switched host to the ancestors of modern humans.

We hypothesise that by combining fossil data on when and where different hominin species were likely to be present in Africa, the geographical range of modern chimpanzees and bonobos, and the reconstructed distribution of tropical rainforest habitat as a proxy for the past range of the chimpanzee/bonobo ancestor (anc-chimps) it will be possible to develop a model to statistically infer the species that facilitated the host-switch of HSV2 in the modern human lineage defined here as beginning with \textit{Homo erectus}~\citep{Anton2016}. Before \textit{Homo erectus} it is unclear which hominins are the direct ancestors of anatomically modern humans. However, once HSV2 has reached \textit{Homo erectus}, no further host-switches are required for HSV2 to be considered present in the ancestors of modern humans.

HSV2 is found in all living populations~\citep{Looker2015}, is accepted as having an African origin~\citep{Koelle2017, Burrel2017}, and has patterns of genetic divergence consistent with having diverged with human populations as they spread out of Africa~\citep{Koelle2017}. Current genetic, archaeological and fossil evidence suggests 100 KYA as a plausible (although not universally accepted) earliest date for anatomically modern humans (AMH) to have left Africa ~\cite{MirazonLahr2016}. We therefore infer that HSV2 must have been in the population of AMH before they left Africa, in order to take it with them when migrating to the rest of the world.

\section*{Methods}

The process of identifying the most probable intermediary species that transmitted proto-HSV2 (henceforth HSV2) from anc-chimps to modern humans involves four steps: (a) data acquisition, which involves collecting information on the prevalence of HSV2 in Africa, fossil records of potential intermediary hominins, their geographical range and time-period, and the extent of tropical rainforest data, (b) network generation, where a network of possible transmission paths of HSV2 from anc-chimps to humans through different hominins is created, (c) probabilistic network analysis, which involves assigning weights to different transmission paths on the graph network based on a certain probability distribution, and (d) optimal path traversal, which evaluates the shortest path from anc-chimps to modern humans based on the probabilistic weights assigned to different paths.

\subsection*{HSV2 prevalence data, hominin fossil data and chimpanzee and tropical rainforest geographic range data}

HSV2 is most closely related to ChHV1 which infected the ancestors of modern chimpanzees. The GIS data on the range of modern \textit{Pan troglodytes} and \textit{Pan paniscus},  provided by the IUCN Red List~\citep{Oates2008}, is shown in~\cref{fig:chimpanzee-rainforest}.  Only one ancestral chimpanzee fossil, dated to c. 500 KYA, is currently known~\citep{McBrearty2005}. This means that the habitat range of anc-chimps is not directly measurable from the fossil record but is inferred to be a larger geographical range than that of modern Pan. We can use the GIS data on the range of modern Pan troglodytes and Pan paniscus, provided by the IUCN Red List (Oates et al. 2008), shown in Figure 1, to make some inferences about the range of anc-chimps, but a measure of paleo-tropical rainforest range is likely to be a better proxy for the range of anc-chimps~\cite{Elton2008}. Therefore, we have used the paleo-tropical rainforest range during the period of 1.4 - 3 MYA from the K\"{o}ppen-Geiger climate classification dataset~\cite{Peel2007} as a proxy for the ancient range of chimpanzees and combined this with data based on modern great ape distribution patterns and range size estimates~\citep{MyersThompson2003}. This is shown in~\cref{fig:chimpanzee-rainforest}. 


To identify which species could have been involved in the host-switch of HSV2 from the ancestors of chimpanzees to the ancestors of modern humans, we collated spatio-temporal data on African hominin fossil species extant between 100 KYA and 3 MYA. Latitude and longitude of site location was used to provide a spatial data point for each species. Temporally, published fossil dating evidence was used to provide a first appearance datum (FAD) and last appearance datum (LAD) for each species (see supplementary Table 1).

Data on the prevalence of herpes simplex virus 2 between 2000 and 2015 CE was taken from the supplementary materials of~\citep{Looker2015}, and plotted to demonstrate the distribution of HSV2 across Africa (Supplementary Figure A. 1). 

\subsection*{Network analysis}
To establish the most probable transmission path of HSV2 from anc-chimps to modern humans, it is important to identify potential species that may have been intermediary hosts. Initially, all hominins in Africa with temporal ranges within the 1.4 - 3 MYA confidence window~\citep{Wertheim2014} of the chimp-hominin transmission were identified (see Supplementary Table 1). Their distance to ancient tropical rainforest was calculated, and only those species whose fossil remains were found within 400 km of tropical rainforest were considered as putative species for initial ancestral-chimp-hominin HSV2 transmission. This initial threshold on the distance reflects the distance that would have been covered by hominins employing three possible strategies: (a) a broadly omnivorous subsistence strategy based on scavenging, (b) hunting using carnivore and herbivore movement patterns and (c) modern hunter-gatherer range sizes~\citep{Foley1978,Grant1992}. A matrix of spatio-temporal distances was then calculated to map the distances between the nearest neighbours of each species and also to calculate the temporal overlap between species using the fossil record (see supplementary Tables 2 and 3). Fossils from four genera (\textit{Ardipithecus}, \textit{Kenyanthropus}, \textit{Orrorin} and \textit{Sahelanthropus}) were excluded from the analysis on the basis that there is no fossil evidence that they persisted after 3 MYA.

A  network of possible transmission paths of HSV2 from anc-chimps to humans through different hominins was developed as a directed acyclic graph (DAG) $G = (V, E)$ comprising of a set of nodes ($V$), representing potential intermediary hosts, and edges ($E$) connecting the nodes, which represents the direction of transmission between species (See~\cref{fig:dag-ip,fig:dag-it}). The DAG comprised of the anc-chimps as the start node, H. erectus as the target node, and other potential species forming secondary nodes in the graph. The edges are typically weighted; in this analysis, the weights are based on the inverse probability of transmission.

If HSV2 was transmitted to \textit{Homo erectus}, no further cross-species transmission event is needed to explain the infection of modern humans. Simple vertical mother-to-child or horizontal (sexual) transmission of the virus through the genus \textit{Homo} from this point would be sufficient as the ancestor-descendent path from \textit{Homo erectus} to \textit{Homo sapiens} is relatively secure~\citep{Maslin2015}.

\subsubsection*{Bayesian inference}
A Bayesian network or a belief network is a graphical structure which allows us to represent and reason about an uncertain domain. In a Bayesian network, the species (nodes) are variables, and the transmission paths (edges) represent direct links between species. The process of conditioning (also called probability propagation or inference) is performed via a ``flow of decisions'' through the network, which involves computing the posterior probability distribution for a set of query nodes (hominins which could have transmitted HSV2), given values for some evidence (or observation) nodes (anc-chimps and \textit{H. erectus}). The presence of HSV2 in the common ancestors of modern humans, and anc-chimps being the source species of the virus, provides the evidence of the presence of HSV2 in anc-chimps and \textit{H. erectus}, and these species form the observation nodes in the belief network. The process of conditioning involves computing the probability of different nodes (hominins) for the acquisition and transmission of HSV2 through all possible transmission paths in the network, such that the observations of HSV2 in anc-chimps (origin node) and \textit{H. erectus} (target node) are true. An important consideration in all Bayesian-based methods is the choice of a prior. An empirical Bayesian method that estimates the likelihood of HSV2 infection using a prior beta distribution is adopted, which is an approximation to a hierarchical Bayesian approach~\citep{Farine2015,Murphy2012}.

 
Bayesian networks provide full representations of probability distributions over their variables, which allows us to infer upon any subset of variables. A Bayesian network is created using the DAG described above. Each node on the graph represents a potential intermediary host that has a probability of transmitting HSV2. The probability of infection transmission is represented as a beta distribution, with shape parameters \textit{alpha} representing the time period of the species in 100,000 years and \textit{beta} representing the distance to the neighbour in kilometres. A \textit{Conditional Probability Table} (CPT) is generated for all possible combinations of a dichotomous outcome for each variable (HSV2 infection is true or false).

For example, to evaluate the Conditional Probability Table of node \textit{Paranthropus boisei} we identified its parent nodes on the network graph as the following species: anc-chimps, \textit{A. africanus}, \textit{H. habilis}, and \textit{H. rudolfensis}. To calculate the probability of transmission from anc-chimps to \textit{P. boisei} we assume a beta distribution. The parameter $\alpha$, in the beta distribution, represents the time-period overlap between the two species, in this case 700 Kyr and $\beta$ represents the distance between the two species (35 kilometers between the fossils of \textit{P. boisei} and the extent of anc-chimps range). Based on this beta distribution, Monte Carlo simulations are run to sample from the distribution. This yields a probability of \textit{P. boisei} acquiring HSV2 from anc-chimps of 0.954. Similarly, the probability of other hominins infecting \textit{P. boisei} are evaluated for \textit{A. africanus}, \textit{H. habilis} and \textit{H. rudolfensis} as 0.096, 0.996 and 0.34, respectively.  A Conditional Probability Table is constructed for the node \textit{P. Boisei} (see Supplementary Table 4).

Conditional Probability Tables are evaluated for each node on the graph network. The Bayesian inference analysis was performed on the network using the AI space (\url{http://aispace.org/bayes/}) decision network tool~\citep{poole2010artificial}. The ``\textit{Combined Inference}'' approach is adopted to evaluate the probability of intermediary hosts transmitting HSV2, by conditioning the anc-chimp and \textit{Homo erectus} nodes for the presence of HSV2~\citep{Korb2003}, i.e. they are treated as being infected by HSV2.

\subsubsection*{Optimal path traversal}

A* is an informed search algorithm that searches through all possible paths to the target that yields the smallest cost~\citep{Hart1968}, which allows us to identify the most probable HSV2 transmission path. This is done by combining information by favouring vertices that are close to the starting point and to the target. At each time-step the A* algorithm selects the path at a given vertex $n$ that has the lowest cost $f(n) = g(n) + h(n)$, where $g(n)$ represents the exact cost of the path from the starting point to any vertex $n$, and $h(n)$ represents the heuristic estimated cost from vertex $n$ to the goal. A* balances the two as it moves from the starting node to the target node. The most probable transmission path is evaluated by minimising the traversal costs based on the edge cost and nodal heuristics of the transmission network graph. A Python script with the \textit{NetworkX} package was used to implement the A* pathfinding algorithm on the DAGs to identify the most probable (optimal) HSV2 transmission path (see \url{https://gist.github.com/wadhamite/e8991aff7d8d3a48d5bc846c747de6d8}).

Two species, anc-chimps and \textit{Homo erectus}, formed the start node and the target node of the DAG. The edges represent the direction of HSV2 transmission and are weighted based on the inverse probability of transmission between the species (nodes). The weights (edge costs) are determined based on two probability models: Infection Prevalence (HSV2-IP) and Infection Transmission (HSV2-IT) based on the temporal and geographic range of the species. More details about the probability models are discussed in the next section. A Conditional Probability Table (CPT) is used to determine the nodal heuristics.
 

\subsubsection*{HSV2-Infection Prevalence (HSV2-IP) model}
The Infection Prevalence model is a local model of probability, which assumes each transmission path and species to be independent: the probability of a species being infected by HSV2 is affected by how long the species persisted and where the species was located, not the probability that other species were infected. 

The model uses a beta distribution to determine the probability of a species transmitting / being infected by HSV2 based on the proximity of the species to the rainforest habitat and the duration (difference between the first and the last appearance datum) of the species. Both the probability of transmission and infection are described using beta distributions. The probability density function for a beta distribution is defined as:  

\begin{equation}
PDF: \frac{x^{\alpha - 1}(1 - x)^{\beta - 1}}{B(\alpha, \beta)} \qquad \mathrm{where} \qquad B(\alpha, \beta) = \frac{\Gamma(\alpha)\Gamma(\beta)}{\Gamma(\alpha + \beta)}
\end{equation}

Where, $\Gamma$ defines a gamma distribution function, $\alpha$ is the time period of existence of the species in 100,000 years and $\beta$ is the spatial distance of the fossil from the rainforest in kilometres. The probability of transmission through a particular path is determined as the combined probability of the species (nodes) forming the edge. For example, the probability of transmission path from anc-chimps ($P(A)$) to \textit{P. boisei} ($P(B)$) is the combined probability of both these species ($P(A*B)$). The probability of transmission from anc-chimps is evaluated using a beta distribution with as 6000 (corresponding to 6 Million years of existence), and  as 100 (distance to the rainforest in km). The beta density functions of HSV2 transmission / infection for anc-chimps and \textit{P. boisei} are shown in Supplementary Figure A. 1. The probability of \textit{P. boisei} being infected also follows the same beta-distribution with the parameters as 700 (corresponding to 700,000 years of existence), and  as 35 (distance to the rainforest in km). The edge costs were estimated by Monte-Carlo simulations of the combined probabilities of two species (nodes), forming the edge, by sampling from their respective beta distributions. A typical run of a Monte-Carlo simulation is shown as a blue line in Supplementary Figure A. 1, where the probability of transmission by anc-chimps ($P(A) = 0.97$) and the probability of \textit{P. boisei} being infected ($P(B) = 0.9375$), evaluates the combined probability of the transmission path, for that run, as 0.909.  Monte-Carlo simulation results are averaged over 1000 simulation runs. Monte-Carlo simulations evaluating combined probabilities $P(A*B)$ give an edge cost of 0.937 for the edge (path) connecting anc-chimps to \textit{P. boisei}, i.e., the probability of HSV2 being transmitted from anc-chimps to \textit{P. boisei} is 0.937.
 

\subsubsection*{Stochastic modelling of infection transmission}
Infection transmission in epidemiology can be predicted using mathematical models: a popular approach is the Susceptible - Infection - Recovered (SIR) model~\citep{Chen2008}. In the stochastic version of the SIR model, the continuous variables are replaced by discrete numbers, and the process rates are replaced by process probabilities. At time $t$ the probability that a new susceptible host is infected is modelled as an exponential distribution, which is epidemiologically incorrect for most diseases~\citep{Wearing2005,Bailey1975,sartwell1950distribution}, i.e., the rate of leaving the exposed is independent of the time spent on the host. \citet{Wearing2005} suggested a more realistic distribution of latent and infectious periods, with a stronger central tendency:

\begin{table*}[!h]
	\centering
	\renewcommand{\arraystretch}{1.5}
	\begin{tabulary}{\linewidth}{LCCCC}
		\# of infectives at time ($t$) & =  & 
		Initial number of infectives who are still infectious at time ($t$) & + & 
		Those who have acquired the infection in the time interval $[0, t]$ and are still infectious at time ($t$)			\\ 
	\end{tabulary}
\end{table*}

More realistic distributions can be obtained by choosing  probability density function of the infectious period,  p(t) to be a gamma probability density function~\citep{Blythe1988,Lloyd2001}.

\subsubsection*{Infection Transmission (HSV2-IT) model}
The proposed Infection Transmission model considers the history of transmission, and the probability of paths (edges) and species (nodes) are dependent on the parent nodes and paths. The probability of a parent node being infected with HSV2 affects the probability of a child node being infected with HSV2 in addition to the proximity and time period overlap between the parent and child nodes. 

The model utilises the temporal overlap between hominin species and their geographic proximity to one another to determine the probability of a transmission path, using a gamma distribution.  The probability density function of a gamma distribution is given as:

\begin{equation}
PDF:\frac{\beta^\alpha}{\Gamma(\alpha)}x^{\alpha - 1} \exp^{-\beta x}
\end{equation}
in terms of shape $\alpha$ and rate $\beta$. The shape parameter $\alpha$ is defined as the ratio of the time period in 1000 years / distance in kilometres and the rate parameter $\beta$ is defined as the normalised time period $Y / x$, where $x$ is the time period of the species in 1000 years and $Y$ is the time period of anc-chimps. The probability density distribution of transmission of HSV2 from anc-chimps is shown in Supplementary Figure A. 3. Monte-Carlo simulations were performed to evaluate the conditional probability of transmission between species, as mutually exclusive events, considering the parent nodes by sampling from the respective probability distributions. 

For example, the probability of \textit{P. boisei} transmitting HSV2 to \textit{H. erectus} depends on the probabilities of \textit{P. Boisei} being infected by the anc-chimps and/or \textit{H. habilis} and/or \textit{H. rudolfensis}. The probability of transmission follows the gamma distribution. The gamma distribution for anc-chimp is defined with a shape parameter $\alpha$ of 6, and a rate parameter $\beta$ of 10. Similarly, the gamma probability distribution is evaluated for other parent nodes of \textit{P. boisei} (i.e., \textit{H. habilis} and/or \textit{H. rudolfensis}) based on their time-period overlap, and distance. The path from \textit{A. afarensis} is ignored in this analysis due to its low probability of transmitting HSV2. A conditional probability of transmitting HSV2 from \textit{P. boisei} to \textit{H. erectus}, given the probability distributions of all possible paths capable of infecting \textit{P. boisei} that is $P(C | A \cup B \cup C)$, is evaluated as 0.152.

\subsection*{Sensitivity analysis}
Sensitivity analysis is performed to identify the impact of the probability distribution models in estimating the edge costs and in turn the optimal path of transmission. A variance-based global sensitivity analysis was performed using the Saltelli method, a variation of the Sobol technique, to evaluate the sensitivity indices of the transmission model~\citep{Sobol2001,Saltelli2010}. The sensitivity of each input is represented by a numeric value, called the sensitivity index. These indices are used to estimate the influence of individual variables or groups of variables on the model output, by decomposing the variance of the model output into fractions. While spatial and temporal range is often presented as a hard number in biology the complexities found on the ground are much more nuanced both in terms of daily interactions and evolutionary trajectories ~\citep{Benton1996}. These complexities are, naturally, further compounded when dealing with a fragmentary fossil record.  In order to allow for natural uncertainty to be introduced into the models the level of variation for sensitivity analysis was set at 10\% to mirror diffusion between temporal and spatial borders. A parametric space of edge costs was generated for the Sobol analysis by sampling 50,000 times from the probabilistic distribution for each edge. This generated a total of 2.3 million analyses for the sensitivity analysis for each transmission model. The first-order, second-order and the total-order indices were measured using a Python script with the \textit{SALib} package.

\section*{Results}

The results from the Bayesian inference and the optimal path traversal using the Infection Prevalence and Infection Transmission models are presented below, beginning with the Bayesian inference analysis, the optimal path traversal results and finally the sensitivity analysis of the different models.

\subsection*{Bayesian inference}
By conditioning anc-chimps and H. erectus for the presence of HSV2, the combined Bayesian inference of the DAG reveals \textit{A. afarensis} (90\%), \textit{H. habilis} (67.8\%), \textit{P. boisei}  (50\%), and \textit{H. rudolfensis} (55.8\%) to be the likely intermediary hosts for the transmission of HSV2. These are the most probable hosts to have acquired HSV2, and a transmission path through these hosts is more likely.

\subsection*{Optimal path traversal}
The optimal traversal path of the DAG for HSV2 transmission was modelled using the Infection Prevalence and the Infection Transmission models. The A* algorithm using the weighted DAGs from both models identified \textit{Paranthropus boisei} as the most probable intermediary hosts that transmitted HSV2 from anc-chimps to the ancestors of modern humans (see~\cref{fig:dag-ip}). In addition to the direct transmission of HSV2 from anc-chimps to \textit{P. boisei}, the Infection Transmission model also identified Homo habilis as an intermediary host that transmitted HSV2 to \textit{P. boisei}, which subsequently transmitted to the ancestors of modern humans (see~\cref{fig:dag-it}).~\Cref{table:hsv2} shows the probable HSV2 intermediary paths and their rankings based on the edge weights from the probabilistic models. 

\begin{table}[!tbhp]
\caption{Probable HSV2 transmission routes}
\centering
\renewcommand{\arraystretch}{1.5}
\newcolumntype{C}[1]{>{\centering\let\newline\\\arraybackslash\hspace{0pt}}m{#1}}
\label{table:hsv2}
\begin{tabular}{l C{0.22\textwidth} C{0.22\textwidth}}
\toprule
Disease Transmission Route & 
HSV2-IP route ranking
(normalised path cost) & 
HSV2-IT route ranking
(normalised path cost) \\ 
\midrule
anc-chimp $\Rightarrow$ \textit{P. boisei} $\Rightarrow$ \textit{H. erectus} & \textbf{1 (1.0)} & 3 (1.14)\\


anc-chimp $\Rightarrow$ \textit{H. habilis}  $\Rightarrow$ \textit{P. boisei} $\Rightarrow$ \textit{H. erectus} & 4 (1.13) & \textbf{1 (1.0)} \\

anc-chimp $\Rightarrow$ \textit{H. habilis} $\Rightarrow$ \textit{H. erectus} & 2 (1.20) & 2 (1.05) \\

anc-chimp $\Rightarrow$ \textit{H. rudolfensis} $\Rightarrow$ \textit{H. erectus} & 3 (1.24) & 4 (1.54) \\

\bottomrule
\multicolumn{3}{l}{* The most probable route has a cost of 1. The path costs are normalised to the shortest path.}
\end{tabular}
\end{table}

\subsection*{Sensitivity analysis}
We performed sensitivity analysis allowing the input parameters of the probabilistic distributions for both the models to vary by 10\%. A total of 2.3 million analyses were performed for each model by sampling 50,000 times from the respective probability distributions for each edge. The cost of transmission of HSV2 along all possible paths (path costs) from anc-chimps to \textit{H. erectus} were calculated. The path cost represents the ease of transmitting HSV2 through the path. A path cost is calculated as an arithmetic sum of the inverse probability of transmission through each edge in a given transmission path. The most probable transmission path is always the one that has the lowest cost (i.e., the sum of the probabilities of all of the edges in the path is the highest). 

\subsubsection*{HSV2-IP model}
The input parameters to this model were: distance to the rainforest and the overlap in the time-period was allowed to vary by 10\%. Sensitivity analysis of HSV2-IP model always predicted the transmission route as `anc-chimps $\rightarrow$ \textit{P. boisei} $\rightarrow$ \textit{H. erectus}'.  The average path cost was 0.587, and it varied between 0.526 and 0.649. The distribution of path costs obtained from the sensitivity analysis is shown in~\cref{fig:beta}.  The total-order and first-order Sobol indices of the `anc-chimp $\rightarrow$ \textit{boisei}’ path is 4.53E-2 and 4.53E-2, respectively. The `\textit{P. boisei} $\rightarrow$ \textit{H. erectus}' path has a total index of 9.55E-1 and a first-order index of 9.55E-1. All other paths had negligible or zero index values. The frequency of occurrence of different transmission paths from the sensitivity analyses are summarised in Supplementary Table 5. When the critical path, defined as the path that costs the least to traverse, through \textit{P. bosei} is not available, the transmission path from anc-chimps to \textit{H. erectus} was through H. habilis (normalised path costs, defined as the ratio of the path cost to the critical path cost in the graph, 1.08 - 1.32 of the critical path cost) for 67\% of the cases and through \textit{H. rudolfensis} (normalised path costs of 1.11 - 1.32 of the critical path cost) for the remaining 33\%.

\begin{figure}[!htbp]
	\centering
	\includegraphics[width=\textwidth]{figs/dag-ip}
	\caption{the A* shortest path route(s) for the Infection Prevalence model. The lines with arrows are the possible transmission paths. The values on the lines are the edge costs (inverse probability of transmission). This model predicts that the host switch of HSV2 occurred through the route ancestral-chimps, \textit{P. boisei} and \textit{H. erectus}. The route remained unchanged in the sensitivity analysis.}
	\label{fig:dag-ip}   
\end{figure}

\subsubsection*{HSV2-IT model}
In this analysis, the input parameters of the gamma distribution were: the proximity of species and the time-period overlap between species, varied by 10\%. The distribution of individual path costs used to populate the edge costs of the DAG used in the sensitivity analysis are shown in~\cref{fig:gamma-paths}. Sensitivity analysis predicted a primary transmission path (60\% of the cases) of lysis predicted a primary transmission route (60\% of the cases) of `anc-chimps $\rightarrow$ H. habilis $\rightarrow$ P. boisei $\rightarrow$ H. erectus` and a secondary transmission path (remaining 40\%) through  `anc-chimps $\rightarrow$ P. boisei $\rightarrow$ H. erectus`. The distribution of the shortest path costs for both the primary and secondary transmission paths are shown in~\cref{fig:gamma}. The Sobol indices for the HSV2-IT model are presented in~\cref{table:sobol}. 

\begin{table}[!tbhp]
\caption{Sobol indices of transmission paths for HSV2-IT model using gamma distribution}
\centering
\renewcommand{\arraystretch}{1.5}
\label{table:sobol}
\begin{tabulary}{\linewidth}{LCC}
\toprule
Disease Transmission Route & 
Sobol total-order indices & 
Sobol first-order indices \\ 
\midrule
anc-chimp $\Rightarrow$ \textit{P. boisei} & 0.640 & 0.454\\


anc-chimp $\Rightarrow$ \textit{H. habilis} & 0.519 & 0.334 \\

\textit{P. boisei} $\Rightarrow$ \textit{H. erectus} & 0.0262 & 0.0262 \\
\bottomrule
\end{tabulary}
\end{table}

  

\begin{figure}[!tbhp]
  \centering
  \includegraphics[width=\textwidth]{figs/dag-it}
  \caption{the A* shortest route for the Infection Transmission model. The lines with arrows are the possible transmission paths. The values on the lines are the edge costs (inverse probability of transmission). This model predicts that the host switch of HSV2 occurred through the primary route of ancestral-chimps, \textit{H. Habilis}, \textit{P. boisei} and \textit{H. erectus}. Sensitivity analysis revealed the primary route occurred 60\% as opposed to the secondary route of ancestral-chimps, \textit{P. boisei} and \textit{H. erectus} (which occurred the remaining 40\%).}
  \label{fig:dag-it}   
\end{figure}     

\begin{figure}[!htbp]
  \centering
  \includegraphics[width=\textwidth]{figs/chimpanzee-rainforest}
  \caption{
  	Map showing the distribution of extant chimpanzee (Pan troglodytes) and bonobo (Pan paniscus) populations [IUCN redlist http://maps.iucnredlist.org/map.html?id=15933, http://maps.iucnredlist.org/map.html?id=15932]; the locations of hominin fossils [supplementary table 1] are shown with markers. The colour of the marker indicates the hominin genus; the symbol represents the species. The map also shows the location of hominin fossils relative to ancient minimum and maximum rainforest distributions~\citep{Peel2007}. This figure is available interactively: \url{https://wadhamite.github.io/hsv2-map/}{https://wadhamite.github.io/hsv2-map/}.}
  \label{fig:chimpanzee-rainforest}
\end{figure}

\begin{figure}[!htbp]
  \centering
  \includegraphics[width=\textwidth]{figs/beta_path_costs}
  \caption{Path costs distribution for Infection Prevalence model. The shortest path is ‘anc-chimp $\Rightarrow$ \textit{P. boisei} $\Rightarrow$ \textit{H. erectus}’ with an average path cost 0.587.}
  \label{fig:beta}   
\end{figure}     



\begin{figure}[!htbp]
  \centering
  \includegraphics[width=0.9\textwidth]{figs/gamma_individual_path_costs}
  \caption{ Distribution of selected edge costs for Infection Transmission model. 
Each histogram shows the distribution of edge costs for the IT model of HSV2 transmission. Colour contours indicate the number of occurrences of a path cost in the sensitivity analysis. The lowest edge costs are seen for ‘anc-chimp $\Rightarrow$ \textit{H. habilis}’, ‘anc-chimp $\Rightarrow$ \textit{P. boisei}’ and ‘\textit{P. boisei} $\Rightarrow$ \textit{H. erectus}’.}
  \label{fig:gamma-paths}   
\end{figure}     



\begin{figure}[!htbp]
  \centering
  \includegraphics[width=\textwidth]{figs/gamma_path_costs}
  \caption{Path costs distribution for Infection Transmission model. Each histogram shows the distribution of path costs for the IT model of HSV2 transmission. Colour contours indicate the number of occurrences of a path cost in the sensitivity analysis. The optimal transmission route is ‘anc-chimp $\Rightarrow$ \textit{H. habilis} $\Rightarrow$ \textit{P. boisei} $\Rightarrow$ \textit{H. erectus}’. (a) Total distribution of path costs for HSV2-IT model, (b) distribution path costs for the primary transmission route (‘anc-chimp $\Rightarrow$ \textit{H. habilis} $\Rightarrow$ \textit{P. boisei} $\Rightarrow$ \textit{H. erectus}’), and (c) distribution path costs for the  secondary transmission route (‘anc-chimp $\Rightarrow$ \textit{P. boisei} $\Rightarrow$ \textit{H. erectus}’).}
  \label{fig:gamma}   
\end{figure}     


\section*{Discussion}
\subsection*{Transmission of HSV2}
Transmission of HSV2
The combined inference analysis of a Bayesian Network, considering the presence of HSV2 in both anc-chimps and Homo erectus, revealed \textit{A. afarensis} (90\%), \textit{H. habilis} (67.8\%), \textit{P. boisei}  (50\%), and \textit{H. rudolfensis} (55.8\%) to be the likely intermediary hosts. Although, \textit{A. afarensis} had a close proximity to the geographical range of anc-chimps, it could only transmit HSV2 to \textit{A. africanus}. However, \textit{A. africanus} has a mere 3\% chance of being infected by HSV2 due its geographic location. This limits the possibility of HSV2 transmission to \textit{H. habilis}, \textit{P. boisei} and \textit{H. rudolfensis}, either directly or as intermediary hosts. 

The Infection Prevalence (HSV2-IP) is a local model that describes the probability of HSV2 infection based on the proximity to the rainforest habitat and the duration a species persisted in the fossil record using a beta distribution. In a beta distribution, the parameters alpha (time period) and beta (proximity to the rainforest) are weighted equally, i.e., an increase in the distance to the rainforest by 1 kilometer has the same effect on probability as a decrease of 1000 years in the time period. Thus the beta distribution assumes a constant rate of infection transmission. The HSV2-IP model considers each transmission as an independent event, i.e., the probability of a species transmitting HSV2 is independent of previous events leading to the HSV2 infection of that species. 

The A* algorithm using the weighted DAG with the HSV2-IP model predicted the transmission of HSV2 from anc-chimps to \textit{H. erectus} through \textit{P. boisei} with an average path cost of 0.587. Sensitivity analysis did not show any variation in the route (i.e., varying the location and the time-period of the probable species by 10\% did not affect the optimal route). Non-zero values of sensitivity indices were observed only along the critical transmission route of `anc-chimp $\rightarrow$ P. boisei $\rightarrow$ \textit{H. erectus}'. Although, the `\textit{P. boisei} $\rightarrow$ \textit{H. erectus}' path accounted for 95.5\% variation in the path costs, the path cost of `anc-chimp $\rightarrow$ \textit{P. boisei}' was the critical value that controlled the transmission route. Hence, the optimal route through \textit{P. boisei} remained unaffected. Further analysis of the route revealed a significant influence (88 - 90\% of the variation in the results) of the proximity and duration of \textit{P. boisei} in contrast to the probability of anc-chimps and \textit{H. erectus} on the path costs. The infection prevalence model uses a beta distribution, which assumes equal weights for the shape parameters: \textit{alpha} and \textit{beta}. As expected, Sobol analysis showed equal influence for the distance and time-period parameters (a first-order Sobol indices of 0.493 and 0.492 respectively) on the path costs. Due to the nature of beta distribution, care should be taken in defining the rate of transmission (ratio of the shape parameters, alpha and beta), as the probability value estimated from the distribution is dependent on the ratio of the shape parameters. The intermediary host identified as \textit{P. boisei} is only 35 kilometers from the rainforest, which results in high probability of transmission, and hence results in the lowest cost of transmission through this species. HSV2-IP is a local model, which is independent of other events and does not consider the history of transmission and the associated probabilities.

The Infection Transmission (HSV2-IT) model assigns probability values (edge costs) to the DAG utilising the temporal overlap between hominin species and their geographic proximity to one another. The HSV2-IT model is history dependent, i.e., the probability of a child node transmitting a disease depends on the probability of the parent nodes infecting the child node in a DAG. The Infection Transmission model predicts an initial transmission of HSV2 from anc-chimps to \textit{H. habilis}. Further, \textit{H. habilis} transmitted the virus to \textit{P. boisei}, which then infected \textit{H. erectus}. The average shortest path cost was estimated as 1.05. Sensitivity analysis of the HSV2-IT model revealed two transmission routes: (a) Primary transmission route (60\% of the cases) of `anc-chimps $\rightarrow$ \textit{H. habilis} $\rightarrow$ \textit{P. boisei} $\rightarrow$ \textit{H. erectus}' and (b) a secondary transmission route (remaining 40\% of the cases) through `anc-chimps $\rightarrow$ \textit{P. boisei} $\rightarrow$ \textit{H. erectus}'. However, the distribution of path costs through both the routes were similar, showing that both transmission routes could have been possible. Sensitivity analysis shows that the `anc-chimp $\rightarrow$ \textit{H. habilis}' and `anc-chimp $\rightarrow$ \textit{P. boisei}' have quite a significant effect on the transmission route (total path cost). About 70\% the variation in the route of `anc-chimp $\rightarrow$ \textit{H. habilis}' route is due to the direct effect of edge weight determined by the gamma distribution in comparison to 64\% of variation for `anc-chimp $\rightarrow$ P. boisei' route. Secondary interactions (interactions between different paths) accounts for 30 - 35\% of the overall routing. The probability of a species transmitting, and another being infected had similar effects (similar Sobol indices), which explains that the model did not have a bias either toward the proximity or the temporal overlap between species. Because the HSV2-IT model considers the history of transmission, it is a better predictor than the HSV2-IP model.

Our analysis suggests that \textit{Paranthropus boisei} was the most critical intermediate host for transmitting HSV2 between anc-chimps and the ancestors of \textit{Homo sapiens}. The transmission of proto-HSV2 from anc-chimps to \textit{P. boisei} could have happened directly or through \textit{Homo habilis}. However, our analysis did not predict \textit{Homo habilis} transmitting HSV2 directly to \textit{Homo erectus}. Our results suggest the initial transmission to \textit{P. boisei} could have happened from \textit{H. habilis} 60\% of the time due (most likely through direct sexual contact rather than consumption of infected tissue) and the remaining 40\% was directly transmitted from anc-chimps via hunting or scavenging.

\textit{P. boisei} would have been well placed to act as an intermediate host for HSV2. It most likely contracted the infection through hunting or more likely scavenging infected ancestral-chimpanzee meat. Processing (with or without tools) and consumption of raw meat would act as a simple route for ChHV1 to have crossed into \textit{P. boisei} via open cuts or sores. Tropical refugia during hot dry periods may have driven chimpanzees into higher concentrations in certain areas, driving them into contact and competition with \textit{P. boisei} and \textit{H. habilis} as the margins of tropical forest blended into more open savannah like habitats~\citep{Julier2017}. Violent confrontation or hunting/scavenging and butchery practices would have provided a viable route of transmission for HSV2. Homo habilis remains have been recovered from the same layers as stone tools and bones carrying evidence of butchery, supporting a possible transmission-through-hunting/scavenging hypothesis for the initial anc-chimp to \textit{H. habilis} transmission~\citep{Clarke2012}. \textit{Paranthropus aethiopicus}, \textit{P. boisei}, and \textit{P. robustus} are associated with the Oldowan stone tool complex~\citep{DeHeinzelin1999}, and \textit{P. boisei} explicitly with butchery~\citep{Dominguez-Rodrigo2013} lending support to the hypothesis that bushmeat hunting/scavenging and butchery may have led to the initial transmission of HSV2 to the hominins. 

Both \textit{Homo erectus} and \textit{Paranthropus boisei} are known from sites around Lake Turkana in Kenya that are contiguous in age~\citep{Anton2016, Wood2007} and it is likely that close contact between the species would have been relatively common especially around water sources. The appearance of \textit{Homo erectus} 2.0 MYA is accompanied by evidence of active hunting and butchery, and from 1.76 MYA increasingly sophisticated stone tools~\citep{Cachel1998}. This behavioural shift towards active hunting displayed by \textit{Homo erectus}, combined with direct archaeological evidence of hunting, provides a credible route for direct  transmission of HSV2 from \textit{P. boisei} to \textit{H. erectus} through contact and/or consumption of infected material processed from \textit{P. boisei} carcasses.

In this study, \textit{Homo erectus} is chosen as the target node for the transmission of HSV2 for a number of reasons. Morphological adaptation for bipedal locomotion is used as the primary trait for assigning fossils to the hominin sub-family, especially between 7.0 - 4.5 MYA, but is not an effective tool for determining patterns of ancestor-descendant relationships in the fossil record. The hominin sub-family currently contains seven genera but the exact taxonomic relationship between each genus is not clearly definable because of the fragmentary nature of the fossil record. Similarly, patterns of intra- and inter-species variation are difficult to define~\citep{Foley2016}.  The adaptive radiation of the genus \textit{Australopithecus} between 4.5 - 2.0 MYA represents the first morphologically coherent group of fossil hominin species but its relationship to the genus Homo is not yet clear. The appearance of \textit{Homo erectus} circa 2.0 MYA in East Africa marks the first appearance of recognisably 'human' morphology, life history and brain development and represents a secure most recent common ancestor (MRCA) for all subsequent Homo species (with the possible exception of \textit{Homo floresiensis}~\citep{Argue2017, Anton2016}. Therefore, the ancestor-descendant relationship between \textit{Homo erectus} at c. 2.0 MYA and \textit{Homo sapiens} c. 200 KYA is an evolutionarily secure route of transmission for HSV2 to leave Africa as a modern human-borne virus.

Although ChHV1 causes outbreaks of oral and pharyngeal lesions in chimpanzees in a manner similar to HSV1, in hominins contracting proto-HSV2, the oral niche was already occupied by HSV1. This may have protected the hominin first infected with HSV2: pre-existing infection with HSV1 reduces the likelihood that subsequent infection with HSV2 will be symptomatic~\citep{Langenberg1999}, and also reduces the risk of HSV2 meningitis (Aurelius et al., 2012). HSV2 may have been forced to adapt to a different mucosal niche in order to reduce competition from the co-evolved, native HSV1. However, both simplexviruses remain capable of infecting both the oral and genital niche in modern humans~\citep{Kim2006,Whitley2007}, and of causing co-infection in both niches (eg 10\% to 15\% of herpes labialis (oral lesions) is caused by HSV-2~\citep{Glick1999}).

We suggest that the mode of transmission of HSV2 to hominins was most likely through hunting injuries (eg chimpanzee bites or cuts sustained during meat processing), although onwards transmission into the ancestors of \textit{Homo sapiens} could have been sexual (horizontal) or a result of hunting injuries (vertical). There are many reports of transmission of B virus (\textit{Cercopithecine herpesvirus} 1), the cercopith homolog of HSV1 and ChHV1, to humans, where disease ranges from mild to fatal. Transmission has occurred from bites and scratches, needle sticks and even scratches from cage bars that are contaminated with B virus-positive bodily fluids~\citep{Huff2003}. Onwards transmission between humans has been reported to occur~\citep{CDC1987}. HSV1 can infect other primates, from gorillas~\citep{Gilardi2014} to owl monkeys~\citep{Melendez1969}, typically causing fatal disease in species more distantly related to \textit{Homo sapiens}, while causing oral lesions and milder disease in great apes such as \textit{Gorilla beringei graueri}~\citep{Gilardi2014}. We therefore infer that herpes simplex-like viruses spread relatively easily between individuals even across species barriers, increasing the chances of transmission between hominins and other primates from close contact such as hunting, butchery, inter-personal violence or sexual contact. Evidence for close hominin-hominid contact is also found in other `\textit{heirloom}' human pathogens~\citep{Houldcroft2017a}.

The high prevalence of HSV2 in central and eastern Africa~\citep{Looker2015} (Supplementary Fig.1 and interactive maps at \url{https://wadhamite.github.io/hsv2-map}) is consistent with the limited genetic data available from African HSV2 isolates. A study from Burrel and colleagues~\citep{Burrel2015} showed that HSV2 can be divided into African and worldwide lineages on the basis of diversity in gene UL30. Furthermore, two groups found evidence of gene flow from HSV1 into HSV2, and speculated that the flow of HSV1 loci into the worldwide HSV2 lineage may have helped this lineage of HSV2 to further adapt to human hosts, and so spread more successfully around the world from around 41 KYA~\citep{Burrel2015,Koelle2017}. Recent studies have significantly increased the number of whole HSV2 genomes available for analysis, contributing to our knowledge of HSV2 diversity~\citep{Kolb2015,Szpara2014}; but there is conflicting evidence on whether the most basal HSV-2 genotypes are from west and central~\citep{Burrel2017} or east~\citep{Koelle2017} Africa. Our analyses predict that individuals from east Africa are likely to carry the most ancient HSV2 lineages. 

The time-depth of ancient DNA analysis is dually limited by technology and preservation of DNA. Similarly, the archaeological and fossil records suffer from differential rates of preservation~\citep{Allentoft2012} and gaps that can never be filled because the material has simply not survived. Our analysis has allowed the reconstruction of hominin/human-disease interaction well beyond the horizon of ancient DNA and at a level that is invisible to the fossil and archaeological records. There are other ancient human pathogens that have switched between different primate and hominin hosts over the last 6 million years and these transmission routes could be further explored with this methodology. For example, human pubic lice (\textit{Pthirus pubis}) were introduced by an unknown hominin through contact with the ancestor of gorillas around 3.3 MYA~\citep{Reed2007}. There is also evidence of Neanderthal to human transmission of human papillomavirus genotypes~\citep{Pimenoff2017}, and of hominin to human transmission of body louse genotypes~\citep{Reed2004a}. The studies and the findings presented here demonstrate the potential for using modern disease genetics to understand better the evolutionary interaction between humans and disease in deep time.

\section*{Data presentation}
The distribution maps were created with custom JavaScript codes using Leaflet.js library  [\url{http://leafletjs.com/}] and interactive maps can be accessed at \url{https://wadhamite.github.io/hsv2-map}

\section*{Author contributions}
SJU and CH conceived the study and contributed data. SJU, KK and CH performed the analyses. SJU, KK and CH wrote the paper. All authors approved the publication of the manuscript.

\section*{Acknowledgements}
The authors would like to thank C Ruis (University College London), and JB Ramond and RF Rifkin (University of Pretoria) for helpful discussion. SJU was funded by Oxford Brookes University. KK and CH were funded by the University of Cambridge. KK is a college research associate at King’s College, Cambridge. CH is a post-doctoral affiliate at Christ’s College, Cambridge.

\printbibliography

\clearpage
\section*{Supplementary material}
\renewcommand\thefigure{A.\arabic{figure}}    
\setcounter{figure}{0} 
\renewcommand\thetable{A.\arabic{figure}}    
\setcounter{table}{0} 

\begin{figure}[!h]
	\centering
	\includegraphics[width=\textwidth]{figs/fossils}
	\caption{Map showing the prevalence of HSV2 in Africa, using data from Looker (2015). The locations of hominin fossils [supplementary table 1] are shown with markers. The colour of the marker indicates the hominin genus; the symbol represents the species. 
This figure is available interactively: \url{https://wadhamite.github.io/hsv2-map/}}
	\label{fig:hsv2}   
\end{figure}  

\clearpage
\centering
\textbf{Table A.1.} Fossil hominins with the potential to acquire - HSV2 from ancestral chimpanzee\\
\renewcommand{\arraystretch}{2}
\tymin=50pt
\tymax=200pt
\begin{ltabulary}{L|CCCL}
	
	\toprule
	\textbf{Species} &  \textbf{Type specimen} & 	\textbf{Locations (type specimen)} &  \textbf{Age (000s BP)} & \textbf{Publication (suggesting a new species)}\\ 
	\midrule
	\endfirsthead
	
	\midrule
	\textbf{Species} &  \textbf{Type specimen} & 	\textbf{Locations (type specimen)} &  \textbf{Age (000s BP)} & \textbf{Publication (suggesting a new species)}\\ 
	\midrule
	\endhead
		 
	% Footer to say continued on next page
	\midrule
	\multicolumn{5}{r}{Continued on next page}\\
	\midrule
	\endfoot
	
	% Footer on the last page of the table
	\bottomrule
	\endlastfoot
	

	\textit{Australopithecus afarensis} & LH4 (mandible) &
	East Africa (Laetoli, Tanzania) & 3900 - 2800 & 
	Johanson DC, White TD and Coppens Y 1978 A new species of the genus Australopithecus (Primates: Hominidae) from the Pliocene of eastern Africa. \textit{Kirtlandia 28: 2-14.}\\
	
	
	\textit{Australopithecus africanus} & Taung 1 (cranium) &
	Southern Africa (Taung, SA) & 2800 - 2300 &
	Dart RA 1925 Australopithecus africanus: The man-ape of South Africa. \textit{Nature 115: 195-199.}\\
	
	
	\textit{Australopithecus gahri} & BOU-VP-12/130 (cranium) &
	East Africa (Bouri, Ethiopia) & 2500 &
	Asfaw B, White T, Lovejoy O, Latimer B,. Simpson S and Suwa G 1999 Australopithecus garhi: A new species of early hominid from Ethiopia. \textit{Science 284: 629-635.}\\
		
	

	\textit{Paranthropus aethiopicus} & Omo 18-1967-18 (mandible) &
	East Africa (Omo, Kenya) & 2700 - 2300 &
	Arambourg C and Coppens Y 1967 Sur la d\'{e}couverte, dans le Pl\'{e}istoc\`{e}ne inf\'{e}rieur de la vall\'{e}e de l’Omo (Éthiopie), d’une mandibule d’australopith\'{e}cien. \textit{Comptes Rendus de l’Acad\'{e}mie des Sciences, Paris, series D, 265: 589-590.}\\
	
	
	\textit{Paranthropus robustus} (including \textit{Paranthropus crassidens}, SK 48) &
	TM 1517 (various) &	Southern Africa (Kromdraai, SA) &	1800 - 1000 &
	Broom R 1938 The Pleistocene anthropoid apes of South Africa. \textit{Nature 142: 377-379.}\\
	
	
	\textit{Paranthropus boisei} & OH 5 (cranium) & 
	East Africa (Olduvai Gorge, Tanzania) & 1750 - 1200 &
	Leakey LSB 1959 A new fossil skull from Olduvai. \textit{Nature 184: 491-493.}\\
	
	
	\textit{Homo habilis} (Australopithecus?) & OH 7 (mandible) & 
	East Africa (Olduvai Gorge, Tanzania) & 2500 - 1600 & 
	Leakey LSB, Tobias PV and Napier JR 1964 A new species of the genus Homo from Olduvai Gorge, Tanzania. \textit{Nature 202: 308-312.}\\
	
	
	\textit{Homo rudolfensis} (Kenyanthropus?) & KNM-ER 1470 (cranium) & 
	East Africa (Koobi Fora, Kenya) & 2400 - 1800 &
	Leakey RE 1973 Evidence for an advanced Plio-Pleistocene hominid from east Rudolf, Kenya. \textit{Nature 242: 447-450.} \\
	
	
	\textit{Homo erectus} & KNM-ER 992 (mandible) & 
	Africa (Koobi Fora, Kenya) & 2000 - 600 & 
	Groves CP and Mazák V 1975 An approach to the taxonomy of the Hominidae: Gracile Villafranchian hominids of Africa. \textit{Casopis pro Mineralogii Geologii 20: 225-247.} \\

\end{ltabulary}

\end{document}
